operators/jaeger/1.26.0/jaeger.v1.26.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,515 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "jaegertracing.io/v1",
+          "kind": "Jaeger",
+          "metadata": {
+            "name": "jaeger-all-in-one-inmemory"
+          }
+        }
+      ]
+    capabilities: Deep Insights
+    categories: Logging & Tracing
+    certified: "false"
+    containerImage: docker.io/jaegertracing/jaeger-operator:1.21.2
```

Comment:
Recent versions imported here use quay.io instead of docker.io (although it seems both should work).  Either way, the tag needs to be updated here.